vexim/adminaliasaddsubmit.php

```diff
@@ -53,7 +53,10 @@
     header("Location: adminalias.php?badname={$_POST['localpart']}");
     die;
   }
-
+  if(!filter_var($_POST['smtp'], FILTER_VALIDATE_EMAIL)) {
+    header ("Location: adminalias.php?invalidforward=".htmlentities($_POST['smtp']));
+    die;
+  }
   $aliasto = preg_replace("/[', ']+/", ",", $_POST['smtp']);
```

Comment:
As you can see here and in `adminaliaschangesubmit.php`, we seem to support multiple comma-separated forward addresses. `filter_var()` would fail these.